### PR TITLE
Sapc4c

### LINKED ISSUE #546: New Element: SAP C4C
## Description
Element documentation for new Element, SAP C4C

### PR DESCRIPTION
**Pull request by davewilks**
## Highlights
* New CRM element
* New Helpdesk element

## Closes
* Closes #546

### CODE REVIEW

File: docs/elements/sapc4c-crm/authenticate.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        "name": "<AUTHENTICATED_INSTANCE_NAME>"
+    }
+    ```
+1. Call the following, including the JSON body you constructed in the previous step:
+
+        POST /instances
+
+    {% include note.html content="Make sure that you include the User and Organization keys in the header. See <a href=index.html#authenticating-with-cloud-elements>the Overview</a> for details. " %}
+
+1. Locate the `token` in the response and save it for all future requests using the element instance.
```
Comment (claytonshaver) [inline docs/elements/sapc4c-crm/authenticate.md]: We may want to include keeping a reference to the instance id as well.

File: docs/elements/sapc4c-hd/authenticate.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        "name": "<AUTHENTICATED_INSTANCE_NAME>"
+    }
+    ```
+1. Call the following, including the JSON body you constructed in the previous step:
+
+        POST /instances
+
+    {% include note.html content="Make sure that you include the User and Organization keys in the header. See <a href=index.html#authenticating-with-cloud-elements>the Overview</a> for details. " %}
+
+1. Locate the `token` in the response and save it for all future requests using the element instance.
```
Comment (claytonshaver) [inline docs/elements/sapc4c-hd/authenticate.md]: Same as above

File: docs/elements/sapc4c-hd/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+{% include callout.html content="<strong>On this page</strong></br><a href=#element-details>Element Details</a></br><a href=#base-url>Base URL</a></br><a href=#authenticating-with-cloud-elements>Authenticating with Cloud Elements</a></br><a href=#error-codes>Error Codes</a>" type="info" %}
+
+# Element Details
+
+| Element Information | Details     |
+| :------------- | :------------- |
+| API Documentation | [{{page.heading}} API  documentation](http://help-legacy.sap.com/saphelp_sapcloudforcustomer/en/PUBLISHING/IntegrationServices.html) |
+| Authentication | Basic  |
+| Events | Polling |
+| Bulk | Supported |
```
Comment (claytonshaver) [inline docs/elements/sapc4c-hd/index.md]: What are your thoughts on including rate limits in this matrix?

Comment (davewilks) [inline docs/elements/sapc4c-hd/index.md]: I think it's a great idea. However, I looked through their docs and couldn't find anything about it. If you take a look at the bulk file, you'll see I made a vague reference to checking with SAP about rate limits.

**Pull request merged**